crates/matrix-sdk-indexeddb/src/crypto_store/mod.rs

```diff
@@ -827,13 +827,25 @@ impl_crypto_store! {
 
         let mut result = Vec::new();
         for _ in 0..limit {
-            result.push(self.deserialize_inbound_group_session(cursor.value())?);
+            result.push(cursor.value());
```

Comment:
Can `result` hold a more meaningful name, now that it's a transient vector? (also bonus since you're around, you can initialize it with `Vec::with_capacity(limit)` instead of `Vec::new` to avoid a few reallocs)

Reply:
Yes, updated https://github.com/matrix-org/matrix-rust-sdk/pull/2969/commits/97e3fd43c7e6b10061a9cf3ba56f7e6d9c3830d6